rest/model/task.go

```diff
@@ -466,13 +466,15 @@ type APISyncAtEndOptions struct {
 }
 
 type APIDependency struct {
-	TaskId string `bson:"_id" json:"id"`
-	Status string `bson:"status" json:"status"`
+	TaskId       string `bson:"_id" json:"id"`
+	Status       string `bson:"status" json:"status"`
+	Unattainable bool   `bson:"unattainable" json:"unattainable"`
```

Comment:
This change was unrelated; I was just annoyed that I couldn't use the rest route to see this and had to go into the DB.